src/journal-remote/journal-remote-main.c

```diff
@@ -65,6 +68,7 @@ STATIC_DESTRUCTOR_REGISTER(arg_key, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_cert, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_trust, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_output, freep);
+STATIC_DESTRUCTOR_REGISTER(arg_compression.opts, freep);
```

Comment:
Please declare `compression_args_clearp()` in journal-remote-util.h and use it here.